src/meta/src/hummock/manager/compaction/compaction_group_schedule.rs

```diff
@@ -85,12 +87,26 @@ impl HummockManager {
             .cloned()
             .collect_vec();
 
+        if member_table_ids_1.is_empty() {
```

Comment:
Is this reachable?

Reply:
yes, we do not remove the default group cg2 and cg3, the dafault group may trigger this branch when initial a new cluster